specification/sql/resource-manager/Microsoft.Sql/preview/2020-11-01-preview/HybridLinks.json

```diff
@@ -0,0 +1,329 @@
+          },
+          "202": {
+            "description": "Accepted"
+          }
```

Comment:
Using 201 + provisioning state + Azure-asyncOperation header is recommended for async PUT.
[https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc0025-201-is-the-recommended-async-pattern](https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc0025-201-is-the-recommended-async-pattern)